lib/Screens/account_dashboard.dart

```diff
@@ -20,97 +25,453 @@ class _AccountDashboardState extends State<AccountDashboard> {
-                                ),
+                              Icon(
+                                icon,
+                                size: 40,
+                              ),
+                              const SizedBox(height: 20),
+                              TitleText(
+                                text: text,
+                                fontSize: 28,
+                                color: textColor,
                               ),
+                              const SizedBox(height: 20),
                               Padding(
-                                padding: const EdgeInsets.fromLTRB(
-                                  12.0,
-                                  6.0,
-                                  12.0,
-                                  12.0,
+                                padding: const EdgeInsets.all(
+                                  8.0,
                                 ),
-                                child: Text(
-                                  accounts[position].accountNumber,
-                                  style: const TextStyle(fontSize: 18.0),
+                                child: ButtonTheme(
+                                  minWidth:
+                                      MediaQuery.of(context).size.width * 0.6,
+                                  height: 70.0,
+                                  child: RaisedButton(
+                                    shape: RoundedRectangleBorder(
+                                      borderRadius: BorderRadius.circular(
+                                        18.0,
+                                      ),
+                                      side: BorderSide(
+                                        color: Colors.grey,
+                                      ),
+                                    ),
+                                    onPressed: () {},
+                                    color: LightColor.navyBlue2,
+                                    textColor: Colors.white,
+                                    child: Text(
+                                      '${t.amount} \$',
+                                      style: const TextStyle(
+                                        fontSize: 28,
+                                      ),
+                                    ),
+                                  ),
                                 ),
                               ),
                             ],
                           ),
                         ],
                       ),
-                      Divider(
-                        height: 2.0,
-                        color: Colors.grey,
+                    ),
+                    const Divider(
+                      height: 8,
+                    ),
+                    Padding(
+                      padding: const EdgeInsets.fromLTRB(
+                        20.0,
+                        0,
+                        20,
+                        0,
```

Comment:
Again, this consumes so many lines, which I think can be combined into a single line since the usage is pretty straightforward...